modules/core/src/v2/keychains.ts

```diff
@@ -33,7 +41,7 @@ Keychains.prototype.get = function(params, callback) {
  * @param callback
  * @returns {*}
  */
-Keychains.prototype.list = function(params, callback) {
+Keychains.prototype.list = function(params: any, callback: NodeCallback<any>) {
```

Comment:
If we don't take any params, and the params object is not used, I think the best way to declare it is `params: {} = {}`. Also callback parameters should always be optional, so please do `callback?:`. Also, missing return type.